ethcore/src/miner/miner.rs

```diff
@@ -263,6 +263,9 @@ impl Miner {
 				let number = block.block().fields().header.number();
 				let difficulty = *block.block().fields().header.difficulty();
 				sealing_work.push(block);
+				if self.work_poster.is_some() {
```

Comment:
Would have preferred to see it closer to the code that relies on it (i.e. the last line of the function).

Good enough for now, but perhaps add a comment saying what it's doing.